From patchwork Tue Oct 13 15:06:51 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Diana Craciun OSS <diana.craciun@oss.nxp.com>
X-Patchwork-Id: 11835797
Return-Path: <SRS0=tiWW=DU=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 9B2A2A5D6
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 13 Oct 2020 18:11:51 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 6A202248D5
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 13 Oct 2020 15:06:59 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1728174AbgJMPGz (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Tue, 13 Oct 2020 11:06:55 -0400
Received: from inva020.nxp.com ([92.121.34.13]:46872 "EHLO inva020.nxp.com"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S1726157AbgJMPGz (ORCPT <rfc822;kvm@vger.kernel.org>);
        Tue, 13 Oct 2020 11:06:55 -0400
Received: from inva020.nxp.com (localhost [127.0.0.1])
        by inva020.eu-rdc02.nxp.com (Postfix) with ESMTP id 6B7761A01DD;
        Tue, 13 Oct 2020 17:06:53 +0200 (CEST)
Received: from inva024.eu-rdc02.nxp.com (inva024.eu-rdc02.nxp.com
 [134.27.226.22])
        by inva020.eu-rdc02.nxp.com (Postfix) with ESMTP id 5EA911A01CC;
        Tue, 13 Oct 2020 17:06:53 +0200 (CEST)
Received: from fsr-ub1864-111.ea.freescale.net
 (fsr-ub1864-111.ea.freescale.net [10.171.82.141])
        by inva024.eu-rdc02.nxp.com (Postfix) with ESMTP id 17F772032C;
        Tue, 13 Oct 2020 17:06:53 +0200 (CEST)
From: Diana Craciun <diana.craciun@oss.nxp.com>
To: alex.williamson@redhat.com, kvm@vger.kernel.org
Cc: linux-kernel@vger.kernel.org, laurentiu.tudor@nxp.com,
        Diana Craciun <diana.craciun@oss.nxp.com>
Subject: [PATCH v2] vfio/fsl-mc: Fixed vfio-fsl-mc driver compilation on 32
 bit
Date: Tue, 13 Oct 2020 18:06:51 +0300
Message-Id: <20201013150651.12808-1-diana.craciun@oss.nxp.com>
X-Mailer: git-send-email 2.17.1
X-Virus-Scanned: ClamAV using ClamSMTP
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

The FSL_MC_BUS on which the VFIO-FSL-MC driver is dependent on
can be compiled on other architectures as well (not only ARM64)
including 32 bit architectures.
Include linux/io-64-nonatomic-hi-lo.h to make writeq/readq used
in the driver available on 32bit platforms.

Signed-off-by: Diana Craciun <diana.craciun@oss.nxp.com>
---
v1 --> v2
 - Added prefix to patch description

 drivers/vfio/fsl-mc/vfio_fsl_mc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/vfio/fsl-mc/vfio_fsl_mc.c b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
index d009f873578c..80fc7f4ed343 100644
--- a/drivers/vfio/fsl-mc/vfio_fsl_mc.c
+++ b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
@@ -13,6 +13,7 @@
 #include <linux/vfio.h>
 #include <linux/fsl/mc.h>
 #include <linux/delay.h>
+#include <linux/io-64-nonatomic-hi-lo.h>
 
 #include "vfio_fsl_mc_private.h"
 
